packages/wp-now/src/start-server.ts

```diff
@@ -0,0 +1,85 @@
+import WPNow from "./wp-now"
+import { HTTPMethod } from "@php-wasm/universal";
+import express from 'express';
+import fileUpload from 'express-fileupload';
+import { DEFAULT_PORT } from "./constants";
+
+
+function generateMultipartFormDataString(json, boundary) {
+  let multipartData = '';
+  const eol = '\r\n';
+
+  for (const key in json) {
+    multipartData += `--${boundary}${eol}`;
+    multipartData += `Content-Disposition: form-data; name="${key}"${eol}${eol}`;
+    multipartData += `${json[key]}${eol}`;
+  }
+
+  multipartData += `--${boundary}--${eol}`;
+  return multipartData;
+}
+
+const requestBodyToString = async (req) => await new Promise((resolve) => {
+  let body = '';
+  req.on('data', (chunk) => {
+      body += chunk.toString(); // convert Buffer to string
+  });
+  req.on('end', () => {
+      resolve(body);
+  });
+});
+
+
+const app = express();
+app.use(fileUpload());
+
+export async function startServer(port:number = DEFAULT_PORT) {
```

Comment:
it might be possible to follow the form of the VSCode WP Playground extension and find the first available port starting with the default port. This can be quite handy when running multiple WordPresses, or when the port is taken.

it didn't require much code - essentially a loop for starting express that's wrapped in a `Promise` which then communicate which port it took.

https://github.com/danielbachhuber/wordpress-playground-vscode/pull/4/files#diff-a4c65ede64197e1a112899a68bf994485b889c4b143198bac4af53425b38406fR18-R25

Reply:
Great suggestion!
I created a `portFinder` singleton so I can know which port is being used from different files:
https://github.com/WordPress/wordpress-playground/pull/188/files#diff-cdfac2859ea11934e3c7b5b8d9cf870ede054e92867f33f7d8d0c6612de8d143